bids_prov/fsl/fsl_parser.py

```diff
@@ -90,7 +80,7 @@ def readlines(filename: str) -> Mapping[str, List[str]]:
             # TODO : add </pre> as in
```

Comment:
Could we store this TODO in an issue to keep track of it?

Reply:
The following issue has been created : https://github.com/Inria-Empenn/BEP028_BIDSprov/issues/25